### PR TITLE
APP-158 [Objective-C] Basic level Apps: Configuration

### PR DESCRIPTION
**Pull request by am1kk**

### CODE REVIEW

**Comment by Slash32** [comment]
@kaa-leeroy add to whitelist

**Comment by Slash32** [comment]
@kaa-leeroy test this please

File: configurationdemo/source/objc/ConfigurationDemo/ViewController.m [hunk 1]
```diff
@@ -91,26 +93,21 @@ - (KAAEmptyData *)getProfile {
...
-    NSMutableString *confBody = [NSMutableString stringWithFormat:@"Configuration body :\n"];
-    for (KAALink *link in links) {
-        [confBody appendString:[NSString stringWithFormat:@"%@ - %@\n", link.label, link.url]];
-    }
-    [self addLogWithText:confBody];
+    KAAConfiguration *configuration = [self.kaaClient getConfiguration];
+    [self addLogWithText:[NSString stringWithFormat:@"Sampling period is %d", configuration.samplingPeriod]];
 }
 
 - (void) addLogWithText:(NSString *) text {
```
Comment (hamsternik) [inline configurationdemo/source/objc/ConfigurationDemo/ViewController.m]: Fix code layout for method signature like so

```
(void)addLogWithText:(NSString *)text {
```

**Comment by hamsternik** [comment]
Please, select methods into _ViewController_ in logical blocks, using `#pragma mark` syntax.

**Comment by Slash32** [comment]
@kaa-leeroy test this please

**Comment by Slash32** [comment]
@kaa-leeroy test this please

**Comment by hamsternik** [comment]
LGTM

**Comment by Slash32** [comment]
@kaa-leeroy test this please

**Pull request merged**